anaconda-gnome-control-center.desktop

```diff
@@ -0,0 +1,16 @@
+[Desktop Entry]
```

Comment:
I wonder if Workstation SIG will be happy about new application in the Gnome Shell like this?
Please take into account that this application will be available also in the installed system.

Reply:
Yes this is a problem, but I dont think there is another way. 
cockpit.spawn did not work, as it's not setting the env variables properly. I can retry implementing this with cockpit-spawn.

Reply:
We should ping Workstation SIG about this. I'll reach them out.